pallets/client-info/Cargo.toml

```
@@ -0,0 +1,42 @@
+[package]
+name = "client-info"
```

Let's maybe stick to interlay's naming of 
```suggestion
name = "clients-info"
```
although for the time being we will only use it for the vault and no oracle or faucet client.

---

We'd also need to adjust the name of the directory and usages in other toml files.